apps/sms/lib/bridge/plugins/dispose/client.js

```diff
@@ -0,0 +1,59 @@
+!((define)=>{define((require,exports,module)=>{
+
+  /**
+   * Logger
+   * @type {Function}
+   * @const
+   */
+  const debug = 1 ? function(arg1, ...args) {
+    var type = `[${self.constructor.name}][${location.pathname}]`;
+    console.log(`[DisposableClient]${type} - "${arg1}"`, ...args);
```

Comment:
nit: I'd remove the quotes around arg1 so that we can also not use a format if we don't want to :)